<launch>
  <remap from="joint_states" to="tilt_motor/joint_states"/>
  <node name="epos_hardware" pkg="epos_hardware" type="epos_hardware_node" args="tilt_joint_motor">
    <rosparam command="load" file="$(find poli_pan_tilt)/config/tilt_controller.yaml" />
  </node>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        output="screen" args="tilt_motor/joint_state_controller tilt_motor/position_controller">
  </node>

  <param name="velocity_controller/type" value="velocity_controllers/JointVelocityController" />
  <param name="velocity_controller/joint" value="tilt_joint" />
  <param name="tilt_motor/position_controller/type" value="position_controllers/JointPositionController" />
  <param name="tilt_motor/position_controller/joint" value="tilt_joint" />
  <param name="tilt_motor/joint_state_controller/type" value="joint_state_controller/JointStateController" />
  <param name="tilt_motor/joint_state_controller/publish_rate" value="20" />

</launch>

